.github/workflows/rust.yml

```diff
@@ -35,6 +35,30 @@ jobs:
           command: test
           args: -p hecs ${{ matrix.features }}
 
+  no-std-build-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          target: x86_64-unknown-none
+          override: true
+
+      # Build the `no_std_macros` example with x86_64-unknown-none target (should not contain std)
+      - uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --no-default-features --features macros --target x86_64-unknown-none --example no_std_macros
+
+      # Build the same `no_std_macros` example with the default target (which should contain std)
```

Comment:
Is this already covered by the generic test job?

Reply:
Yes looks like this is not necessary.

I removed all changes to the workflow config file, as the `no_std` build test is a part of the integration test now.